### ISSUE TITLE
v0.1.0 release.

### ISSUE THREAD

**Issue by songy23** [census-ecosystem/opencensus-java-jdbc#32]
We just made a bunch of updates to this repo and I believe we have fixed all the issues. The artifact name is updated to "io.opencensus.integration:opencensus-jdbc", and the code now complies to Google Java format and error-prone check. Now I think it's the time to cut branch and make a new release to Maven Central.

/cc @odeke-em @bogdandrutu

**Comment by odeke-em** [comment]
Awesome, so we've covered a whole lot of progress in the past couple of days and closed all the issues in the v0.0.2 milestone. I believe this warrants a v0.0.2 release. Once that is out, we can file more issues and tackle them over the next milestone when we have time. What do y'all think @songy23 
 @bogdandrutu?

**Comment by songy23** [comment]
Agree that we have made enough progress for a new patch release. I cannot think of other blockers for this release.

**Comment by bogdandrutu** [comment]
I need one more PR.

**Comment by songy23** [comment]
I think we've made all the necessary changes and ready for a new release now. I would call this release "0.1.0" rather than "0.0.2", because:
1. There's already a v0.0.2 tag on Github: https://github.com/opencensus-integrations/ocjdbc/releases/tag/0.0.2.
2. We renamed the artifact from "io.opencensus" to "io.opencensus.integration", and it could be confusing if we keep using the same minor version, i.e "io.opencensus.ocjdbc:0.0.1" -> "io.opencensus.integration.jdbc:0.0.2".
